components/FAQ.tsx

```diff
@@ -1,50 +1,99 @@
 import React from "react";
 import styles from "../styles/FAQ.module.css";
-import FaqOneQuestion from "./elements/FaqOneQuestion";
+import Image from "next/image";
+
+function CloseIcon(props: any) {
+	return (
+		<div>
+			<span className={styles.closeIcon}>
+				<Image
+					src="/icons/faqClose.png"
+					alt="Free Medical Advice"
+					width={25}
+					height={25}
+					layout="fixed"
+					className={styles.iconClass}
+				/>
+			</span>
+		</div>
+	);
+}
 
 export default function FAQ() {
 	return (
 		<div className={styles.globalFaq} id="faq">
 			<h2>Frequently Asked Questions</h2>
 			<div className={styles.faqArray}>
-				<FaqOneQuestion
-					question={
-						"How long does it take to hear back from a medical doctor?"
-					}
-					answer={
-						"You should hear back from a healthcare provider on the Medex platform in a few minutes after request. If your request was sent at night, then the response time may increase to serval hours of waiting"
-					}
-				/>
-				<FaqOneQuestion
-					question={"Who is Medex for?"}
-					answer={
-						"Medex is a digital health service that provides telemedicine and telepharmacy for adults to treat their health problems quickly and effectively at any time and any place."
-					}
-				/>
-				<FaqOneQuestion
-					question={"Who is the health care professionals in Medex?"}
-					answer={
-						"All of the healthcare team on the Medex platform are licensed healthcare professionals. Each doctor undergoeas an extensive background check and license verification process. When you message or talk with a physican on the Medex platform you can see their resume and medical license."
-					}
-				/>
-				<FaqOneQuestion
-					question={"Can I use any pharmacy?"}
-					answer={
-						"Yes, you can just choose the address of your local pharmacy and medications will be shipped there for free."
-					}
-				/>
-				<FaqOneQuestion
-					question={"Can I purchase medications with home delivery?"}
-					answer={
-						"OTC medications: Yes, Rx medications: No, The current position of the Chief Pharmaceutical Inspector is identical to the act and is clear - internet - order sale of prescription drugs is impossible. Pharmaceutical law regulations in force in Poland do not allow this."
-					}
-				/>
-				<FaqOneQuestion
-					question={"Is my personal data safe?"}
-					answer={
-						"Yes. We take your privacy and the security of your information very seriously. Please read our Privacy Policy for more details. "
-					}
-				/>
+				<div className={styles.faqBox}>
+					<CloseIcon />
```

Comment:
- [ ] kolejna rzecz ten komponent FaqOneQuestion był lepszym rozwiązaniem
- [ ] kolejna rzecz h4 to zły nagłówek dla odpowiedzi na pytania
- [ ] kolejna rzecz cała sekcja faq powinna być html-owym &lt;secion opatrzona
- [ ] a czy to serio taki problem, żeby zaimplementować zamykanie? jest kilka poziomów zamykaja, 
  - ( ) najprostszy to użycie usestate osobnego dla każdej sekcji, 
  - ( ) kolejny to użycie useState([false, false, true, true])  gdzie każdy wpis to stan otwarcia/zamknięcia, wada jest taka, że nie ma enkapsulacji tego komponentu i dochodzi do modyfikowania stanu komponentu będącego kontenerem, a on niekoniecznie musi wiedzieć jaki jest stan okienka z pytaniem
  - ( ) inny sposób to przechowywanie useRef do każdego elementu i operowanie na css display: none/block tego elementu za pomocą `onClick w `CloseIcon`
  - ( ) kolejna rzecz to skorzystanie z Context providera i opatrzenie wszystkich pytań w faq contextem i wsadzenie tego array do tego useContent, polecam dobrze byłoby gdybyś to potrafił zastosować
  - ( ) no i ostatnie to redux/rematchjs, powineneś to znać
  - ( ) * z jakimś higher-order-reducer dla każdej sekcji, wtedy byłbyś pro, powinieneś - sposób z "gwiazdką"